### DISCUSSION TITLE
daydata / monthdata call in crontab

### DISCUSSION THREAD

**Discussion by shadowbyte68** [SBFspot/SBFspot#597] [Q&A]
a log file is written or expanded each time daydata / monthdata is called.
If I turn off the calls to reduce writes to the microSD card, the upload to pvoutput no longer works. Why ?
As far as I understand, the data is taken from the SQL file after all...

**Comment by sbf-** [comment]
> If I turn off the calls to reduce writes to the microSD card

What exactly do you mean with this? Daydata script should run every 5mins to fill the SQL db

**Comment by shadowbyte68** [comment]
Excuse me ! I have my own script for regular reading.
I was surprised that csv files were always written, even though I use the "-nocsv" parameter in my own script.
I then saw the two calls in crontab, which I deactivated. However, the data is then no longer transferred to pvoutput....

**Reply by sbf-** [reply]
Ok, and what is the SBFspot commandline in your script?

**Comment by shadowbyte68** [comment]
here is the command from my script:

`/usr/local/bin/sbfspot.3/SBFspot -v -finq -ad0 -ae0 -am0 -nocsv -sp0 -mqtt`

**Reply by sbf-** [reply]
The -ad0 argument disables the daydata request and also pvoutput export.

**Reply by sbf-** [reply]
-sp0 should be removed too

**Reply by shadowbyte68** [reply]
> -sp0 should be removed too

Hello, I don't want to save any more csv files, so I used "-sp0" and "-nocsv"
So there is no way to completely do without the csv?

**Reply by sbf-** [reply]
Set CSV_export=0 in config file or -nocsv on commandline

**Comment by shadowbyte68** [comment]
Can I combine the calls of "daydata" and "month data" in my 5min script without any disadvantages?

`/usr/local/bin/sbfspot.3/SBFspot` -v -finq -ad1 -ae1 -am1 -nocsv -sp0 -mqtt`

**Reply by sbf-** [reply]
Sure but every call will take a bit more time.
